tool/src/main/java/migt/Main.java

```diff
@@ -184,6 +185,13 @@ public Main() {
 
         readMsgDefFile();
         readConfigFile();
+
+        //--------------------------------------
+        //DRIVER_PATH = Main.class.getClassLoader().getResource("driver/geckodriver").getPath();
+        //DRIVER_PATH = JOptionPane.showInputDialog(null, "enter the driver path:");
+        DRIVER_PATH = "/usr/local/bin/geckodriver";
```

Comment:
I would prefer to not hardcode this path. If this has to do with the docker container we can somehow read it from a ENV variable passed in the Dockerfile, As this basically overwrite the value set in the config file

Reply:
I didn't add it, I think it was used to make testing easier. I'll remove it.